app/src/main/java/com/goldenraven/padawanwallet/data/Repository.kt

```diff
@@ -77,14 +77,14 @@ object Repository {
         if (firstTimeTutorialsDone) {
             editor.putBoolean("firstTimeTutorialsDone", false)
             editor.putBoolean("e1", false)
-                .putBoolean("e2", false)
-                .putBoolean("e3", false)
-                .putBoolean("e4", false)
-                .putBoolean("e5", false)
-                .putBoolean("e6", false)
-                .putBoolean("e7", false)
-                .putBoolean("e8", false)
-                .apply()
+            editor.putBoolean("e2", false)
+            editor.putBoolean("e3", false)
+            editor.putBoolean("e4", false)
+            editor.putBoolean("e5", false)
+            editor.putBoolean("e6", false)
+            editor.putBoolean("e7", false)
+            editor.putBoolean("e8", false)
+            editor.apply()
```

Comment:
Refactoring the `.putBoolean` was purely because the unit tests were having errors on those lines without the editors on them, but I think the scope functions will be much cleaner. I will look into them again and see if theres any changes I can make.